vispy/scene/visuals.py

```diff
@@ -23,26 +23,26 @@
 class VisualNode(Node):
     _next_id = 1
     _visual_ids = weakref.WeakValueDictionary()
-    
+
     def __init__(self, parent=None, name=None):
         Node.__init__(self, parent=parent, name=name,
                       transforms=self.transforms)
         self.interactive = False
         self._opacity_filter = ColorFilter()
         self.attach(self._opacity_filter)
-        
+
         self._id = VisualNode._next_id
         VisualNode._visual_ids[self._id] = self
         VisualNode._next_id += 1
-        self._picking_filter = PickingFilter(id_=self._id)
+        self._picking_filter = PickingFilter(id=self._id)
```

Comment:
Frankly, I'd rather that it would be just `id`, I know its to prevent hiding the buildin `id` function, but this trailing underscore is ugly IMO.